serving/samples/helloworld-nodejs/README.md

```diff
@@ -179,7 +179,7 @@ folder) you're ready to build and deploy the sample app.
 
     ```shell
     curl -H "Host: helloworld-nodejs.default.example.com" http://{IP_ADDRESS}
-    Hello World: NOT SPECIFIED
+    Hello World: Node.js Sample v1
```

Comment:
Should be "Hello Node.js Sample v1!"

Reply:
also please update the corresponding line in the [service.yaml](https://github.com/knative/docs/blob/c1be002db74a3874ba66cd1c2db9e39f00f85fe0/serving/samples/helloworld-nodejs/service.yaml#L15)

Reply:
The `service.yaml` file is correct... the `"Hello "` and `"!"` originates from the `app.js` file...